src/Canvas2d/babylon.canvas2d.ts

```diff
@@ -1688,7 +1688,15 @@
                 this._worldSpaceNode = settings.customWorldSpaceNode;
                 this.applyCachedTexture(null, null);
             }
+
+            this.propertyChanged.add((e, st) => {

```

Comment:
well, that was the original intend of the branch....which slightly derived.
if I do a separated PR it will be on uncompiled code (since i've got TS 2.0 now). but it's a copy/paste... ok...let's try! :)